# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpChannelParent.h
# Commit: d59b42f58f86
# Full Hash: d59b42f58f861e049c1270035ea7568acc0605b9
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P3 send OS*R and ODA via pHttpChannel for multipart channel, r=mayhemer,necko-reviewers
#   
#   Depends on D76969
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D76970
# ==============================================================================

diff -r cbcd3ee87d28 -r d59b42f58f86 netwerk/protocol/http/HttpChannelParent.h
--- a/netwerk/protocol/http/HttpChannelParent.h	Tue Jun 30 00:32:38 2020 +0000
+++ b/netwerk/protocol/http/HttpChannelParent.h	Tue Jun 30 00:32:53 2020 +0000
@@ -356,13 +356,6 @@
   // Used to ensure methods can't be called before OnStartRequest.
   uint8_t mAfterOnStartRequestBegun : 1;
 
-  // Set to true if we get OnStartRequest called with an nsIMultiPartChannel,
-  // and expect multiple OnStartRequest calls.
-  // When this happens we send OnTransportAndData and OnStopRequest over
-  // PHttpChannel instead of PHttpBackgroundChannel to make synchronizing all
-  // the parts easier.
-  uint8_t mIsMultiPart : 1;
-
   // Number of events to wait before actually invoking AsyncOpen on the main
   // channel. For each asynchronous step required before InvokeAsyncOpen, should
   // increase 1 to mAsyncOpenBarrier and invoke TryInvokeAsyncOpen after
